knowledge_base/_Glossary.md

```diff
@@ -0,0 +1,250 @@
+
+See [models/comment.ts](../libs/model/src/models/comment.ts).
+
+### Community
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+A community is a space on Common, joinable by PROFILES, that owns metadata and is defined by a set of ROLES and PERMISSIONS for participation.
+
+See [models/community.ts](../libs/model/src/models/community.ts)
+
+### Core
+
+A set of features considered central to Common, which interface out with first-party apps. As of 240123, the core consists of a SIGN-IN system, a NOTIFICATION system, and a GROUP system.
+
+### Forum
+
+The Common forum is Common's flagship first-party app. Its primary organizational primitive is the COMMUNITY. The forum app provides UI for off-chain discussions, and for interacting with community-associated chains.
+
+### Group
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+Groups are used by community admins to handle permissioning across the forum, using associated membership objects that LINK ADDRESSES to Groups.
+
+See [models/group.ts](../libs/model/src/models/group.ts) and the knowledgebase [Groups](./Groups.md) entry.
+
+### Link
+
+Some content on Common can be inter-linked, so that the content's UI prominently reflects its relationship to other content. As of 240123, THREADS can be linked to other THREADS as well as to PROPOSALS (both Snapshot and on-chain).
+
+### Member
+
+<!-- TODO: Review & update in light of new permissioning system; blocked pending meeting with Product. -->
+
+Generically, a member is a PROFILE that holds a (ROLE) within a COMMUNITY.
+
+More specifically, "Member" is the lowest-ranking (ROLE) in COMMUNITIES, below MODERATOR and ADMIN.
+
+See [models/membership.ts](../libs/model/src/models/membership.ts).
+
+### Notification
+
+*Related Terms:* [Subscription](#subscription)
+
+Notifications are messages emitted to users who are SUBSCRIBED to a given category of activity atop a given forum entity (e.g. a THREAD or COMMUNITY).
+
+See also [models/notification.ts](../libs/model/src/models/notification.ts) and our KB entry [Notifications.md](./Notifications.md). For a list of activity categories which may trigger notification, see [NotificationCategories](../libs/core/src/types.ts).
+
+### Page Scope
+
+The Common website consists of scoped and unscoped pages. Scoped pages, or "community" pages, belong to Common communities; their URI syntax takes the form of e.g. `/scope/discussion/identifier`. Unscoped pages, or "global" pages, are community-independent. Examples include the landing and `/terms` pages.
+
+For URI parsing, route setup, and the scope-conditional rendering of page layouts, see `CommonDomainRoutes.tsx`,  `setupAppRoutes.ts` and `Layout.tsx`.
+
+### Profile
+
+A profile is one or more ADDRESS, grouped together as a single identity, representing a single USER.
```

Comment:
Not quite true -- technically a User consists of multiple Addresses and also a Profile, which specifies social metadata to display on the Profile page. We want to ultimately eliminate the Profile object altogether, as it should be merged with User, but who knows when we'll be able to make that change.